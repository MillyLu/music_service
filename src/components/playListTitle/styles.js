import styled from "styled-components";
import {ReactComponent as WatchSvg} from '../svg/time/watch.svg'

export const ContentTitle = styled.div`
display: flex;
flex-direction: row;
align-items: center;
justify-content: space-between;
margin-bottom: 24px;
`;

export const PlaylistTitleCol = styled.div`
font-style: normal;
font-weight: 400;
font-size: 14px;
line-height: 24px;
letter-spacing: 2px;
color: #696969;
text-transform: uppercase;

`;

export const PlaylistTitleCol01 = styled(PlaylistTitleCol)`
width: 418px;
`;

export const PlaylistTitleCol02 = styled(PlaylistTitleCol)`
width: 200px;
`;

export const PlaylistTitleCol03 = styled(PlaylistTitleCol)`
width: 150px;
`;

export const PlaylistTitleCol04 = styled(PlaylistTitleCol)`
width: 60px;
text-align: end;
`;

export const PlaylistTitleSvg = styled(WatchSvg)`
width: 12px;
height: 12px;
fill: transparent;
stroke: #696969;
`;