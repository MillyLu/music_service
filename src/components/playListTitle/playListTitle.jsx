import * as Styled from './styles';



export function ContentPlaylistTitle(){
    return(
        <Styled.ContentTitle>
            <Styled.PlaylistTitleCol01>Трек</Styled.PlaylistTitleCol01>
            <Styled.PlaylistTitleCol02>ИСПОЛНИТЕЛЬ</Styled.PlaylistTitleCol02>
            <Styled.PlaylistTitleCol03>АЛЬБОМ</Styled.PlaylistTitleCol03>
            <Styled.PlaylistTitleCol04>
                <Styled.PlaylistTitleSvg alt='time'>
                    <use xlinkHref='img/icon/sprite.svg#icon-watch' />
                </Styled.PlaylistTitleSvg>
            </Styled.PlaylistTitleCol04>
        </Styled.ContentTitle>
    )
}