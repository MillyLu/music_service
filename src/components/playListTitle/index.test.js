/* eslint-disable no-undef */
import { render } from "@testing-library/react";

import { ContentPlaylistTitle } from "./playListTitle";

describe('<ContentPlaylistTitle/>', () => {
    it('should render successfully and match the snapshots', () => {
        const {container} = render(<ContentPlaylistTitle/>);
        expect(container).toMatchSnapshot();
    })
})