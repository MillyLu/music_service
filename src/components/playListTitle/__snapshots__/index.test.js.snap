// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ContentPlaylistTitle/> should render successfully and match the snapshots 1`] = `
<div>
  <div
    class="sc-bcXHqe jIIzOc"
  >
    <div
      class="sc-gswNZR sc-dkrFOg dngbGH kjGiDv"
    >
      Трек
    </div>
    <div
      class="sc-gswNZR sc-hLBbgP dngbGH hmpNNA"
    >
      ИСПОЛНИТЕЛЬ
    </div>
    <div
      class="sc-gswNZR sc-eDvSVe dngbGH ltuYS"
    >
      АЛЬБОМ
    </div>
    <div
      class="sc-gswNZR sc-jSUZER dngbGH dGRvtD"
    >
      <svg
        alt="time"
        class="sc-gKPRtg eaFwHt"
      >
        watch.svg
      </svg>
    </div>
  </div>
</div>
`;
